docs/team/skills.md

```diff
@@ -0,0 +1,103 @@
+# Skills and ways to contribute
+
+There are many ways that you can contribute to the JupyterHub / Binder projects.
+This page attempts to provide some pathways for community members to learn and
+apply their skills to the team.
+
+## Where to start?
+
+The short answer to this question is: it depends :-)  there are many
+different ways to contribute to the JupyterHub community - technical, mentorship,
+communications, community work, and documentation are just a few. You can either
+apply your skills to help out, or use it as an opportunity to learn something new!
+
+## A few general ways to help out
+
+* Look through the issues in a repository, and help respond to issues that haven't been addressed already
+* Guide conversation in issues so that they get closer to having a clear, actionable path forward
+* For general questions about using JupyterHub, assist the person and then improve the documentation
+  so that the answer is easier to find.
+* For issues labeled "bug" try to clarify what is going on, and reproduce the bug on your machine
+* For issues with reproducible bugs, in a PR write a test that will fail because of the bug.
+* For issues with reproducible bugs, in a PR fix the bug and make sure that the test passes.
+* For issues that describe new features, engage in conversation in the issue to define an actionable
+  path forward.
+* For issues that describe new features and with a clear path forward, open a PR that implements
+  some or all of the feature.
+
+## Things to know about JupyterHub
+
+This section provides a combination of "skills that are helpful to learn in order
```

Comment:
```suggestion
This section provides a combination of "helpful skills to learn as a contributor"
```

Reply:
The original wording would deter some folks with imposter syndrome as they may not think their skills are strong enough or that they need to know "all" of the skills to participate.